### PR TITLE
[KOGITO-8194] Upgrade Kogito-serverless-operator to operator-sdk 1.25

### PR DESCRIPTION
**Pull request by desmax74**
See: https://issues.redhat.com/browse/KOGITO-8194
Signed-off-by: desmax74 <mdessi@redhat.com>

**Description of the change:**

**Motivation for the change:**

**Checklist**

- [ ] Add or Modify a unit test for your change
- [ ] Have you verified that tall the tests are passing?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -277,4 +277,8 @@ bump-version:
 	./hack/bump-version.sh $(new_version)
 
 install-operator-sdk:
-	./hack/ci/install-operator-sdk.sh
\ No newline at end of file
+	./hack/ci/install-operator-sdk.sh
+
+#docker-buildx Not needed in our operator Operator 1.25.0 https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.25.0/
```
Comment (spolti) [inline Makefile:282]: I would keep it as suggested in the operator-sdk upgrade guide.

File: go.mod [hunk 2]
```diff
@@ -6,7 +6,6 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/jpillora/backoff v1.0.0 // indirect
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
-	github.com/onsi/ginkgo v1.16.5
```
Comment (spolti) [inline go.mod:9]: why did you remove?
shouldn't this dep be updated to 2.1.4?

**Review by radtriste — APPROVED** [review]
ok for me once concerns from @spolti are dealt

File: go.mod (hunk 2 above)
Comment (spolti) [reply go.mod:9]: that's weird, @davidesalerno any hint about what can be happening?

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
@desmax74 btw, the sdk 1.25.1 is released, let's use it, there is no updates needed, just refer it in the readme.
